specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,764 @@
+        .AddText(Text(L"Incoming Call")
+            .SetIsCallScenario(true))
+        .AddImage(Image(LR"(https://unsplash.it/100?image=883)")
+            .SetUsesCropCircle(true))
+        .AddButton(Button(L"Text Reply")
+            .SetIconUri(LR"(Assets/Icons/message.png)")
+            .AddArgument(L"action", L"textReply")
+            .AddArgument(L"callId", 938163))
+        .AddButton(Button(L"Reminder")
+            .SetIconUri(LR"(Assets/Icons/reminder.png)")
+            .AddArgument(L"action", L"reminder")
+            .AddArgument(L"callId", 938163))
+        .AddButton(Button(L"Ignore")
+            .SetIconUri(LR"(Assets/Icons/cancel.png)")
+            .AddArgument(L"action", L"ignore")
+            .AddArgument(L"callId", 938163))
+        .AddButton(Button(L"Answer")
+            .SetIconUri(LR"(Assets/Icons/telephone.png)")
+            .AddArgument(L"action", L"answer")
+            .AddArgument(L"callId", 938163))
+    .Show();
+```
+
+Example result:
+
+![AppNotification IncomingCall Example](AppNotificationIncomingCallExample.png)
+
+If the developer wants the XML payload to add more components, AppNotificationContent::GetXml will
+return the current version of their AppNotificationContent in a string format.
+
+```c#
+winrt::hstring xmlPayload{ AppNotificationContent()
+    .AddArgument(L"launch")
+        .AddText(Text(L"Content"))
+    .GetXml() };
+```
+
+AppNotificationContent also has a Show API that allows the developer to display an AppNotification
+of what has been added so far to the AppNotificationContent.
+
+```c#
+winrt::hstring xmlPayload{ AppNotificationContent()
+    .AddArgument(L"launch")
+        .AddText(Text(L"Content"))
+    .Show() };
+```
+
+# Text
+
+The Text component sets up the xml for a \<text\> element. Developers can define the content,
+language, and placement of the text on the AppNotification.
+
+**WinAppSDK 1.2 \<text\> Schema**:
+
+```c#
+<text lang? = string
+      placement? = "attribution"
+      hint-callScenarioCenterAlign? = boolean />
+```
+
+These attributes are abstracted with the Text component APIs:
+
+-   SetLanguage(String language)
+-   SetIsAttributionText(Boolean isAttributionText)
+-   SetIsCallScenario(Boolean isCallScenario)
+
+Below are some example usages:
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Content")
+        .SetLanguage(L"en-US"))
+    .AddText(Text(L"Attribution Text")
+        .SetIsAttributionText(true))
+    .Show();
+```
+
+![Text Example](TextExample.png)
+
+Developers can use SetIsCallScenario(bool) to center the text like an incoming call. This feature
+will be ignored unless the AppNotification scenario is set to IncomingCall using
+AppNotificationContent::SetScenario(enum ScenarioType).
+
+```c#
+AppNotificationContent()
+    .SetScenario(ScenarioType::IncomingCall)
+    .AddText(Text(L"Content"))
+        .SetIsCallScenario(true)
+    .Show()
+```
+
+![IncomingCall Example](IncomingCallExample.png)
```

Comment:
A better image here would be the entire notification with the centered text called out